src/core/mixins/userMixin.js

```diff
@@ -44,7 +58,6 @@ export const userMixin = {
         }
         await this.updateAlenviUser(path);
 
-        this.$store.commit('rh/saveUserProfile', this.mergedUserProfile);
         this.emailLock = true;
```

Comment:
Pour regler le soucis dont tu parlais, je pense que tu peux mettre ici `if (path === 'local.email') this.emailLock = true`

Reply:
ouai cool :) j'ai rajouter une condition pour pas permettre de fermer le cadenas si le champs est vide
` if (path === 'local.email' && this.tmpInput !== '') this.emailLock = true;`

Reply:
Tu l'as ajouté ligne 50 mais pour que ca regle le soucis il faut aussi que tu l'ajoutes ligne 61 non ?